app/models/signup.py

```diff
@@ -0,0 +1,80 @@
+import re
+import datetime
+from ..db.db import *
+
+
+class CheckDate:
+    """
+    class contains methods for data validation
```

Comment:
Considering your docstring and the class methods, I would say the name doesn't belong